assets/css/alert-card.scss

```diff
@@ -0,0 +1,95 @@
+.alert-card {
```

Comment:
This is appropriately named -- BUT there's another `alert-card` on the OFT side that is also adding styles.

For example, each alert card has a margin-top of 24px, but there should only be an 8px between cards.

Reply:
Fixed this with a bit of a band-aid. Changed the OFT `.alert-card` to `.oft-alert-card`. You think that's enough or should I find a better solution?

Reply:
This whole file belongs to OFT, so you can nest all these styles (or at least just the alert-card style) in a selector for `outfront-container`, the wrapper for the whole OFT side of the app

Reply:
Good thinking. There are some CSS classes near the bottom of the file that don't directly live in the container so I left them out just in case. They should not conflict with anything in the Screenplay code though.

Reply:
No, those directly live in the container, too. We'll have to build override previews eventually, so in case anything else gets named `pager` or `portrait-png`, let's wrap it all in the selector.